src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -210,23 +233,28 @@ module Read =
             "Mdb{action:l} stream={stream} count={count}/{batches} version={version}",
             action, streamName, count, batches, version)
 
-    let private logLastEventRead streamName t events version (log: ILogger) =
+    let private logLastEventRead (span: Activity) streamName t events (version: int64) (log: ILogger) =
         let bytes = resolvedEventBytes events
         let count = events.Length
         let reqMetric : Log.Measurement = { stream = streamName; interval = t; bytes = bytes; count = count}
         let evt = Log.Metric.ReadLast reqMetric
+        if span <> null then span.AddTag("eqx.bytes", bytes).AddTag("eqx.count", count).AddTag("eqx.version", version) |> ignore
         (log |> Log.prop "bytes" bytes |> Log.event evt).Information(
             "Mdb{action:l} stream={stream} count={count} version={version}",
             "ReadL", streamName, count, version)
 
     let internal loadLastEvent (log : ILogger) retryPolicy (reader : MessageDbReader) requiresLeader streamName
         : Async<int64 * ITimelineEvent<EventBody> array> = async {
+        use span = Tracing.source.StartActivity("ReadLast")
+        if span <> null then
+            span.AddTag("eqx.stream_name", streamName) |> ignore
+            if requiresLeader then span.AddTag("eqx.requires_leader", true) |> ignore
         let! ct = Async.CancellationToken
         let read _ = readLastEventAsync reader streamName requiresLeader ct |> Async.AwaitTaskCorrect
 
         let! t, page = Log.withLoggedRetries retryPolicy "readAttempt" read log |> Stopwatch.Time
 
-        log |> logLastEventRead streamName t page.Messages page.LastVersion
+        log |> logLastEventRead span streamName t page.Messages page.LastVersion
```

Comment:
While `span` is not confusing me and in tracing terms is the correct term... it is a bit overloaded wrt System.Span and/or stream spans, maybe `act` or `activity` makes more sense give that's what it is?